orga/app/components/routes/authenticated/campaigns/new-item.js

```diff
@@ -21,6 +21,13 @@ export default class NewItem extends Component {
     return !this.wantIdPix;
   }
 
+  get targetProfilesOptions() {
+    if (!this.args.targetProfiles) return [];
+    return this.args.targetProfiles.map((targetProfile) => {
+      return { value: targetProfile.id, label: targetProfile.name };
+    });
```

Comment:
En utilisant le map de lodash on peut se passer de cette vérification : 
```suggestion
    return _.map(this.args.targetProfiles, (targetProfile) => {
      return { value: targetProfile.id, label: targetProfile.name };
    });
```

Reply:
Maintenant, dans les toutes les applis js, on privilégie l'utilisation des fonctions natives (ex: map, filter...) plutôt que celle de lodash. Par ailleurs, il ne faut plus utiliser l'import global de lodash mais uniquement les fonctions nécessaires.